cntp-windeploy/main.cpp

```diff
@@ -25,28 +30,67 @@ int main(int argc, char** argv) {
         return 1;
     }
 
+    auto sdkVersion = QStringLiteral("10.0.22000.0");
+    if (parser.isSet("sdk")) sdkVersion = parser.value("sdk");
 
-    QStringList contemporaryPath = Common::findInPaths("contemporary.dll", a.libraryPaths(), true);
-    if (contemporaryPath.isEmpty()) {
+    auto qtPath = qEnvironmentVariable("QT_ROOT_DIR", qEnvironmentVariable("Qt6_DIR"));
+    if (parser.isSet("qt-path")) qtPath = parser.value("qt-path");
+
+    auto hostQtPath = qEnvironmentVariable("Qt6_DIR");
+    if (parser.isSet("host-qt-path")) hostQtPath = parser.value("host-qt-path");
+
+    QStringList libraryPaths{"/Program Files", "/Program Files (x86)"};
+    if (parser.isSet("library-path")) libraryPaths = parser.values("library-path");
+    libraryPaths.append(qtPath);
```

Comment:
Where is this used?

Reply:
CI now installs CMake stuff to a folder instead of Program Files - this allows CI to specify that libraries should be searched for inside that folder instead of Program Files, but keeps compatibility with previous versions if the argument is not set